Library/Formula/facebook-flint.rb

```diff
@@ -0,0 +1,36 @@
+class FacebookFlint < Formula
+  desc "Flint - An open-source lint program for C++ developed by, and used at Facebook."
+  homepage "https://github.com/facebook/flint"
+  url "https://github.com/facebook/flint.git"
+  sha256 ""
+  version "1.0"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "dmd" => :build
+  depends_on "folly" => :build
+  depends_on "gnu-sed" => :build
+  depends_on "libtool" => :build
+  
+  def install
+    ENV.deparallelize
```

Comment:
Can you please submit this issue to the upstream developers of this project and add a link to the upstream issue submission in a comment in the formula file. Thanks!